packages/stark-ui/src/modules/session-ui/pages/login/login-page.component.spec.ts

```diff
@@ -15,7 +19,8 @@ import {
 	MockStarkUserService
 } from "@nationalbankbelgium/stark-core/testing";
 import { TranslateModule } from "@ngx-translate/core";
-import { RawParams } from "@uirouter/core";
+import { StarkAppLogoModule } from "@nationalbankbelgium/stark-ui";
```

Comment:
This import should be relative ;)

Reply:
I thought that is why we have `"@nationalbankbelgium/stark-ui": ["."]` in the tslint.json? 😕

Reply:
True, you're right... my bad :(